examples/simple-regression/src/model.rs

```diff
@@ -0,0 +1,80 @@
+use crate::dataset::DiabetesBatch;
+use burn::{
+    module::Module,
+    nn::{loss::MSELoss, Linear, LinearConfig},
+    tensor::{
+        backend::{AutodiffBackend, Backend},
+        Tensor,
+    },
+    train::{RegressionOutput, TrainOutput, TrainStep, ValidStep},
+};
+use burn::nn::loss::Reduction::Mean;
+use burn::nn::ReLU;
+
+#[derive(Module, Debug)]
+pub struct LinearModel<B: Backend> {
+    input_layer: Linear<B>,
+    output_layer: Linear<B>,
+    activation: ReLU
+}
```

Comment:
I would rename it to `RegressionModel` since it's not technically linear (there is one activation function used).